build/yamls/patches/gke/cniPath.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: antrea
+    component: antrea-agent
```

Comment:
I think these labels are not needed here as kustomization will add them.

Reply:
Done